devx/docker-compose-opensearch.yml

```diff
@@ -0,0 +1,38 @@
+services:
```

Comment:
Was you able to run your integration test with this conf? I wasn't. After some changes (see next comments) got this error `OpenSearch\Common\Exceptions\BadRequest400Exception: {"error":{"root_cause":[{"type":"query_shard_exception","reason":"failed to create query: Engine [NMSLIB] does not support filters","index":"llphant_test","index_uuid":"SHp5NysiQ-eceS8wW-oyIA"}],"type":"search_phase_execution_exception","reason":"all shards failed","phase":"query","grouped":true,"failed_shards":[{"shard":0,"index":"llphant_test","node":"NCDSdG3nSRyz0Sj3zLm4_w","reason":{"type":"query_shard_exception","reason":"failed to create query: Engine [NMSLIB] does not support filters","index":"llphant_test","index_uuid":"SHp5NysiQ-eceS8wW-oyIA","caused_by":{"type":"illegal_argument_exception","reason":"Engine [NMSLIB] does not support filters"}}}]},"status":400}
at vendor/opensearch-project/opensearch-php/src/OpenSearch/Connections/Connection.php:674
at vendor/opensearch-project/opensearch-php/src/OpenSearch/Connections/Connection.php:332
at vendor/react/promise/src/FulfilledPromise.php:28
at vendor/ezimuel/ringphp/src/Future/CompletedFutureValue.php:65
at vendor/ezimuel/ringphp/src/Core.php:341
at vendor/opensearch-project/opensearch-php/src/OpenSearch/Connections/Connection.php:264
at vendor/opensearch-project/opensearch-php/src/OpenSearch/Connections/Connection.php:242
at vendor/opensearch-project/opensearch-php/src/OpenSearch/Transport.php:112
at vendor/opensearch-project/opensearch-php/src/OpenSearch/Client.php:1906
at vendor/opensearch-project/opensearch-php/src/OpenSearch/Client.php:1382
`

Reply:
@f-lombardo No, unfortunately I was not able to run integration tests. I couldn't find the [in the docs mentioned](https://github.com/theodo-group/LLPhant/blob/main/CONTRIBUTING.md#integration-tests) `docker-compose.yaml` file in the `/devx` folder. Therefore I'm unable to setup the required database and run integration tests.

Nonetheless, I updated the `OpenSearchVectorStore` and `OpenSearchVectorStoreTest` files to hopefully resolve this issue. If it doesn't please let me know, how I can setup the environment to run integration tests on my machine.